spring-xd-dirt/src/main/java/org/springframework/xd/dirt/plugins/spark/streaming/SparkStreamingPlugin.java

```diff
@@ -100,20 +102,22 @@ public void postProcessModule(Module module) {
 			}
 			LocalMessageBusHolder messageBusHolder = new LocalMessageBusHolder();
 			messageBusHolder.set(module.getComponent(MessageBus.class));
-			receiver = new MessageBusReceiver(messageBusHolder, storageLevel, messageBusProperties, inboundModuleProperties);
+			receiver = new MessageBusReceiver(messageBusHolder, storageLevel, messageBusProperties,
+					inboundModuleProperties, getMimeType(module, true));
```

Comment:
the meaning of the true and false args here are unclear here. Can we pass ModuleTypeConversionPlugin.INPUT_TYPE and ModuleTypeConversionPlugin.OUTPUT_TYPE here instead?